src/ptp.rs

```diff
@@ -110,9 +112,47 @@ pub trait PtpCommandCode: Sized + Copy {
             *a as u16
         }
     }
+
+    fn enum_name(&self) -> String;
 }
 
-impl PtpCommandCode for StandardCommandCode {}
+impl PtpCommandCode for StandardCommandCode {
+    fn enum_name(&self) -> String {
+        match *self {
```

Comment:
Trying to add the use, but it's not working: 
src/ptp.rs:121:13: 121:32 error: unresolved import `StandardCommandCode::*`. Maybe a missing `extern crate StandardCommandCode`? [E0432]
src/ptp.rs:121         use StandardCommandCode::*;